tutormail/server.py

```diff
@@ -128,7 +128,11 @@ def translate_recipient(self, rcptto):
             return ['mathiasrav@outlook.dk']
         groups = self.get_groups(name)
         if groups:
-            emails = self.get_group_emails(name, groups)
+            emails = []
+            if 'best' in groups:
```

Comment:
This condition will never trigger since `groups` is a list of `str,int`-tuples (name, year). It should check for `("best", self.gf_year)` instead. (This project is in need of some static type annotations.)

Reply:
I expect to rewrite a lot of the infrastructure in the coming period, so I think the lack of type annotations is fine for now. 😛